libsoftwarecontainer/include/container.h

```diff
@@ -124,32 +126,32 @@ class Container :
         return FileToolkitWithUndo::createSymLink(source, destination);
     }
 
-    /*!
-     * Calls shutdown, and then destroys the container
+    /**
+     * @brief Calls shutdown, and then destroys the container
      */
     ReturnCode destroy();
     ReturnCode destroy(unsigned int timeout);
 
-    /*
-     * Calls shutdown on the lxc container
+    /**
+     * @brief Calls shutdown on the lxc container
      */
     ReturnCode shutdown();
     ReturnCode shutdown(unsigned int timeout);
 
-    /*
-     * Calls lxc-stop (force stop)
+    /**
+     * @brief Calls lxc-stop (force stop)
```

Comment:
Sure. I didn't really bother with double checking all the old comments so there might be more problems with the actual comments tbh...